docs/operational_guide/placement_configuration.md

```diff
@@ -64,10 +64,10 @@ The instructions below all contain sample curl commands, but you can always revi
 
 #### Placement Initialization
 
-Send a POST request to the `/api/v1/placement/init` endpoint
+Send a POST request to the `/api/v1/services/m3db/placement/init` endpoint
```

Comment:
I think right now it's a little ambiguous as to when to use `/api/v1/...` and when to use others like `/api/v1/services/m3db/...`. There are other mentions of `/api/v1/placement` like in `docs/how_to` (Github docs) and in our docker integration tests. It would be great to get some clarification on when to use which, but perhaps that's for a larger discussion in another PR.

Reply:
@justinjc It should always be the /services/ ones, we just kept the old names in to not make a breaking change. Anywhere we have a URL without the service name should just be updated to include it